en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md

```diff
@@ -83,7 +83,7 @@ Update the file inside the `Definition` folder with any API related change. The
     - API name (Title)
     - Version
     
-    The version must not be edited once deployed because in the standalone mode, the version represents the identity of the API. Therefore, editing these values will make the API behave as a new API while continuing to keep the previous version of the same API.
+    must not be edited once deployed because in the standalone mode, the version represents the identity of the API. Therefore, editing these values will make the API behave as a new API while continuing to keep the previous version of the same API.
```

Comment:
Must?

Reply:
This is not related to the helm deployment, @suksw can u add a comment here?

Reply:
This is a correction for a typo I made during a previous update. I requested @renuka-fernando to remove the two words because `version` is already present in the bullet points

Reply:
Shall we say something like:

"The following properties must not be edited once deployed:

- API name (Title)
- Version

The version represents the identity of the API when in standalone mode and editing these values will make the API behave as a new API, while continuing to keep the previous version of the same API."

Something like that.

Reply:
+1. That would make it clearer. Looks like there's another typo I missed. Suggesting the exact change above without `The version` at the beginning of the paragraph. Like: 

"The following properties must not be edited once deployed:

    - API name (Title)
    - Version

A combination of these properties represents the identity of the API when in standalone mode and editing these values will make the API behave as a new API, while continuing to keep the previous version of the same API."

Reply:
Thanks, @suksw and @samgnaniah 
fixed with https://github.com/wso2/docs-apim/pull/5214/commits/615cf57a1b30f3e47d512345bbbd470fa15f7888